README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 [![CI](https://github.com/nginx/unit/actions/workflows/ci.yml/badge.svg)](https://github.com/nginx/unit/actions/workflows/ci.yml "GitHub workflow CI")
+[![GitHub Discussions](https://img.shields.io/badge/NGNIX_Unit-discussions-g)](https://github.com/nginx/unit/discussions "GitHub Discussions")
```

Comment:
You have a point. I've updated the color for all three badges, and they seem to more closely match the color of the NGINX logo:

<img width="873" alt="Screenshot 2024-06-05 at 1 58 11 PM" src="https://github.com/nginx/unit/assets/3287976/fd2a96ab-9ac9-4811-96ec-adff9ef9bf72">

Reply:
OTOH, the text in the green part of the badges are a little more difficult to read? (#feEngineeringIsHarderThanItLooks)

Reply:
> For comparison, here's the "current" green in `main`

<img width="664" alt="Screenshot 2024-06-05 at 2 02 00 PM" src="https://github.com/nginx/unit/assets/3287976/5e8df75b-b695-4ec1-a84f-2c4ecc9675b1">

Reply:
@ac000 , I'll go with whatever you prefer.

Reply:
Ooh I like that!

The main problem though is that the 'CI' badge changes dynamically depending on whether the workflows pass or not.

According [to](https://github.com/badges/shields/issues/5437) it may may be doable using <https://shields.io/badges/endpoint-badge>

But I'd want this as two separate patches anyway, so do the first patch that just adds the new badge, using GitHub instead of NGINX Unit for the left-hand side of the badge.

Then we can look at re-colouring the others...

Reply:
> The main problem though is that the 'CI' badge changes dynamically depending on whether the workflows pass or not.

Hi @ac000, I didn't know this. 

So I'll limit this PR to adding the GitHub Discussions badge. Since we want to "re-color" ( ;) ) the others, I'll set up a follow-up issue.